setup.cfg

```
@@ -1,3 +1,76 @@
+[metadata]
+name = siphon
+description = A collection of Python utilities for interacting with the Unidata technology stack.
+description-file = README.rst
```

While a copy/paste of what was there before, I'm pretty sure this needs to be `_` rather than `-`:
```suggestion
description_file = README.rst
```

---

Ah, thank you.